tests/sparsezoo/v2/test_model_directory.py

```diff
@@ -59,6 +60,7 @@
                 "sample_outputs",
                 "sample_labels",
```

Comment:
Just making sure, the presence of `sample_labels` is also integration-dependent, or am I wrong here?